### PR TITLE
VG-3677: Fixes the case where items removed from the cart are still sent for processing.

### PR DESCRIPTION
**Pull request by newfoundgeo**

### CODE REVIEW

**Comment by ryantxu** [comment]
Can one of the admins verify this patch?

File: app/src/taskrunner/task-controller.js [hunk 1]
```diff
@@ -155,7 +133,17 @@ angular.module('taskRunner')
                 query.params.q += ')';
             } else if(hasItems && !_.isEmpty(query.bounds) ) {
                 query.params.q += ' OR (' + query.bounds.replace('&fq=','') + ')';
-            } else if(angular.isDefined(query.solrFilters)) {
+            } else if(angular.isDefined(query.filters)) {
```
Comment (scottlepp) [inline app/src/taskrunner/task-controller.js]: please add a test for this scenario

**Comment by scottlepp** [comment]
hey jason the build is failing for the PR

**Comment by newfoundgeo** [comment]
Should be all good now.

**Comment by newfoundgeo** [comment]
Scott, should be good to go. Please review. Thanks, Jason!

**Pull request merged**